kong/clustering/services/sync/rpc.lua

```diff
@@ -336,6 +338,7 @@ local function do_sync()
   end
 
   if wipe then
+    ngx_log(ngx_INFO, "[kong.sync.v2] full sync ends")
```

Comment:
a blank line to separate this log with other code.

Reply:
done